docs/self-hosted/configuration/environment-variables.md

```diff
@@ -36,17 +46,40 @@ development:
   database: chatwoot_dev
```

Comment:
@vishnu-narayanan  is this section that describes the `database.yml` relevant?

Reply:
No, removed.